gossip/minimal/configpb/config.proto

```diff
@@ -33,16 +34,36 @@ message LogConfig {
   google.protobuf.Duration min_req_interval = 4;
 }
 
+// HubConfig describes the configuration options for a destination
+// Gossip repository, which may be either a Gossip Hub or a regular
+// CT log.
+message HubConfig {
+  // Human-readable name for the hub, must be unique.
+  string name = 1;
+  // Base URL for the hub.
+  string url = 2;
+  // Hub's public key. This is optional, but if omitted signatures from
+  // the log will not be checked.
+  keyspb.PublicKey public_key = 3;
+  // Minimum interval between requests to the hub, for rate limiting.
+  google.protobuf.Duration min_req_interval = 4;
+  // Whether the hub is a true Gossip Hub.
+  bool is_hub = 5;
+}
+
 // GossipConfig describes the configuration of a gossiper.
 message GossipConfig {
   // The source logs whose STHs will be logged.
   repeated LogConfig source_log = 1;
   // The destination hubs to which the minimal-gossip certificates will
   // be submitted.  These destination hubs need to be configured to accept
-  // root_cert as an acceptable root.
-  repeated LogConfig dest_hub = 2;
-  // The root certificate used for submissions, in PEM format; this should
-  // include the public key corresponding to private_key below.
+  // submissions.
+  repeated HubConfig dest_hub = 2;
+
+  // The root certificate used for submissions to any CT logs, in PEM format;
```

Comment:
I prefer the existing wording, as it describes what `gosmin` actually does rather than describing how `gosmin` might fit into a wider ecosystem.  

(I mostly avoid using 'gossip' as a verb in the stuff written so far, because there's no real [gossiping](https://en.wikipedia.org/wiki/Gossip_protocol) going on yet -- just building tools that would form part of a gossip system in future.)